examples/use_case_repository/setup.py

```diff
@@ -0,0 +1,23 @@
+from setuptools import find_packages, setup
+
+setup(
+    name="use_case_repository",
+    packages=find_packages(exclude=["use_case_repository_tests"]),
+    install_requires=[
+        "dagster",
+        "dagster-embedded-elt",
+        "dagster-pipes",
```

Comment:
Just for my understanding, will we need to add a dependency for every library that has a use case generated?

Reply:
I believe so -- this is similar to other example repos. The dependencies are required in order to run tests, e.g. to ensure that types are correct and python is valid syntax.

Reply:
is there a plan to further organize this `use_case_repository` dir to be structured like this?

```bash
/use_case_repository
  /pipes_cli_command
    /setup.py # have its own dependencies, or requirements.py to keep things very simple
    /pipes_cli_command.md
    /pipes_cli_command.py
    /external_script.sh
  /snowflake_to_s3_embeded_etl
    /setup.py # have its own dependencies
    /snowflake_to_s3_embeded_etl.md
    /snowflake_to_s3_embeded_etl.py
```

i could also see this directory eventually replacing the `examples/` directory (i.e. retiring all outdated examples).

Reply:
That's a good point yea, I think once we end up needing separate dependencies, we will have to do that. :(